cli/index-migrate

```diff
@@ -28,6 +28,13 @@ if ((!addressFormat.test(program.contract) && program.state_file === undefined)
 
 return new Promise(async (resolve) => {
     const stateJson = await contractHelper.extractContractFromJSONFile(program.state_file);
+    let config = await contractHelper.extractContractFromJSONFile(program.config);
```

Comment:
I think renaming `extractContractFromJSONFile` to something like `readFromJSONFile` would be more descriptive.

Reply:
Will do!